amp-conformance/CMakeLists.txt

```diff
@@ -1,4 +1,7 @@
-include_directories(${CMAKE_CURRENT_SOURCE_DIR}/amp_test_lib/inc ${LIBCXX_HEADER})
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}/amp_test_lib/inc)
+if (USE_LIBCXX)
+  include_directories(${LIBCXX_HEADER})
```

Comment:
yes, C++AMP conformance was run with OFF on ubuntu 14.04 and with ON on ubuntu 16.06.  Issues exposed by libstdc++ in those tests were also fixed